tools/RAiDER/interpolator.py

```diff
@@ -100,3 +106,27 @@ def fillna3D(array, axis=-1):
     out = dfd.values.reshape(array.shape)
     
     return np.moveaxis(out,-1,axis)
+
+
+def interpolateDEM(demRaster, outLL, extent):
+    ''' Interpolate a DEM raster to a set of lat/lon query points '''
+    from scipy.interpolate import RegularGridInterpolator as rgi
+    minlat, maxlat, minlon, maxlon = extent
+    nPixLat = demRaster.shape[0]
+    nPixLon = demRaster.shape[1]
+    xlats = np.linspace(minlat, maxlat, nPixLat)
+    xlons = np.linspace(minlon, maxlon, nPixLon)
+#    interpolator = RegularGridInterpolator(
+#        grid=(xlats, xlons), 
+#        values=demRaster,
+#        fill_value = np.nan
+#    )
+    interpolator = rgi(
```

Comment:
might be better to track the projection information and use gdal warp with an interpolation scheme. Also see the tropo notebook on ISCE-docs page which has a GDAL interpolation command.